src/napari_spatialdata/_model.py

```diff
@@ -19,19 +18,15 @@ class ImageModel:
     """Model which holds the data for interactive visualization."""
```

Comment:
A bit off-topic for this PR, but maybe I'd rename the `ImageModel` class to `DataModel`

Reply:
lol, this is indeed what I call it in another plugin xD